src/normalizer.ts

```diff
@@ -43,6 +43,30 @@ rules.set('Default top level `id`', (schema, rootSchema, fileName) => {
   return schema
 })
 
+function escape(
+  code: string | { [key: string]: any },
```

Comment:
Call this `schema`, to align with the other rules. Also, its type should just be `JSONSchema`.